linear_decoder.ipynb

```diff
@@ -1,702 +1,900 @@
 {
```

Comment:
<p>let's use the same preprocessing here as we do in the quick start, (subset option) no need for this dense step</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/YosefLab/scvi-tutorials/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>